src/server/src/routes/student.ts

```diff
@@ -73,6 +74,30 @@ router.post('/update-profile', async (req: studentRequest, res: Response) => {
     }
 });
 
+/******************************************************************************
+ *          GET Request - Read - "GET /api/student/get-profile"
+ ******************************************************************************/
+
+router.get('/get-profile', async (req: Request, res: Response) => {
+    const { id } = req.body;
```

Comment:
Same here studentId be received from the user object found in the request (as long as the request comes with a cookie, i.e user is authenticated). Note you would have to use the JWTStudent interface (declared in entities/user.ts). LMK if this makes sense.